lib/helpers/rgba.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  getRgba: getRgba
+};
+
+function getRgba(imageStep) {
+  var rgba = {
+    r: 0,
+    g: 0,
+    b: 0,
+    alpha: 1
+  };
+
+  if (imageStep.red) {
```

Comment:
Per feedback https://github.com/asilvas/node-image-steam/issues/71#issuecomment-468741336 please consider just using a single `background` (`bg` for short) string and translating hex and rgb(a) from that.

Reply:
Alternatively if you wanted to enhance the router logic to support `(x, y, z)` with commas, that would be even cleaner. The need for `(x; y; z)` is just a simpler workaround since it does not require further router changes.

Reply:
Only problem I've found there is you can't use a # in the url

Reply:
Well you can, just encoded. But I'd be good with `hex(123123)` instead. Would probably simplify the parser.

Reply:
OK I've made the changes, not sure how good it is as I'll be honest my regex skill are not the best.